src/application/scene/css/Todo.css

```diff
@@ -0,0 +1,110 @@
+.root {
+    -fx-background-color: #CCCCCC;
+}
+
+.navigation_bar {
+    -fx-background-color: #332E43;
+    -fx-text-fill: #FFFFFF;
+    -fx-border-style: hidden;
+}
+
+.button_tool {
+    -fx-text-fill: #FFFFFF;
+    -fx-background-color: transparent;
+    -fx-font-size: 20px;
+    -fx-font-family: 'Space Mono', monospace;
+    -fx-opacity: 0.6;
+}
+
+.button_tool:hover {
+    -fx-opacity: 0.8;
+}
+
+.button_tool:pressed {
+    -fx-background-color: #5A59B1;
+    -fx-opacity: 1.0;
+}
+
+.button_tool:selected {
+    -fx-background-color: #5A59B1;
+    -fx-opacity: 1.0
+}
+
+.textField_eventName {
+    -fx-prompt-text-fill: black;
+    -fx-border-radius: 4px;
+    -fx-font-size: 20px;
+    -fx-font-family: 'Space Mono', monospace;
+    -fx-opacity: 0.6;
+}
+
+
+.button_addEvent {
+    -fx-background-color: #332E43;
+    -fx-font-size: 20px;
+    -fx-font-family: 'Space Mono', monospace;
+    -fx-opacity: 0.6;
+}
+
+
+.text_listTitle {
+    -fx-font-size: 30px;
+    -fx-font-family: 'Space Mono', monospace;
+}
+
+.listView_TODO {
+    -fx-background-color: #ede7f6;
+
+}
+
+
+.listView_DOING {
+    -fx-background-color: #fff176;
+}
+
+.listView_DONE {
+    -fx-background-color: #81c784;
+}
+
+.list-cell {
+    -fx-font-size: 18px;
+}
+
+.listView_TODO .list-cell:odd {
+    -fx-background-color: #ffffff;
+}
+
+.listView_TODO .list-cell:even {
+    -fx-background-color: #ede7f6;
+}
+
+.listView_DOING .list-cell:odd {
+    -fx-background-color: #ffffa8;
+}
+
+.listView_DOING .list-cell:even {
+    -fx-background-color: #fff176;
+}
+
+.listView_DONE .list-cell:odd {
+    -fx-background-color: #b2fab4;
+}
+
+.listView_DONE .list-cell:even {
+    -fx-background-color: #81c784;
+}
```

Comment:
I find the colours a bit overwhelming here. Given that yellow and green aren't found anywhere else in the app, could they be changed to something more blue/purple toned? I also think the contrast between the odd and even rows is a bit excessive, could it be toned down a little?